doc-surrealdb_versioned_docs/version-2.x/reference-guide/full-text-search.mdx

```diff
@@ -6,98 +6,42 @@ description: In SurrealDB, Full-Text Search supports advanced features like basi
 ---
 
 # Full-Text Search
+In SurrealDB, Full-Text Search supports text matching, proximity searches, result ranking, and keyword highlighting⁠.
+It is also [ACID-compliant](https://en.wikipedia.org/wiki/ACID), which ensures data integrity and reliability.⁠
 
-Full-Text search is a feature that allows search capabilities within a database. This feature can be customized with specific analyzers and tokenizers to refine the search process.
-
-In SurrealDB, Full-Text Search supports advanced features like basic and advanced text matching, proximity searches, result ranking, and keyword highlighting⁠.
-
-Full-Text Search in SurrealDB is [ACID-compliant](https://en.wikipedia.org/wiki/ACID), which ensures data integrity and reliability.⁠
-
-## Core Concepts and Definitions
-
-In order to perform Full-Text Search on textual data within SurrealDB, we leverage analyzers, tokenizers, indexes and filters to optimize the search precision.
-We also use different search functions to highlight keywords and improve their relevance.
+## Define tables and fields
+To implement Full-Text Search it is important that your data is first defined using SurrealDB [tables](/docs/surrealdb/2.x/surrealql/statements/define/table) and [fields](/docs/surrealdb/2.x/surrealql/statements/define/field). 
```

Comment:
I think this might be a bit confusing since `DEFINE TABLE` and `DEFINE FIELD` are used for structure and organisation rather than data definition. Can this sentence be reworded to show that you need data in those fields? 
```suggestion
To implement Full-Text Search it is important that your data is first defined using SurrealDB [`DEFINE TABLE`](/docs/surrealdb/2.x/surrealql/statements/define/table) and [`DEFINE FIELD`](/docs/surrealdb/2.x/surrealql/statements/define/field). 
```